app/code/Magento/Eav/Block/Adminhtml/Attribute/Edit/Options/Options.php

```diff
@@ -145,7 +145,9 @@ protected function _prepareOptionValues(
     ) {
         $type = $attribute->getFrontendInput();
         if ($type === 'select' || $type === 'multiselect') {
-            $defaultValues = explode(',', $attribute->getDefaultValue());
+            $defaultValues = is_string($attribute->getDefaultValue())
+                ? explode(',', $attribute->getDefaultValue())
+                : [''];
```

Comment:
```suggestion
            $defaultValues = is_string($attribute->getDefaultValue())
                ? explode(',', $attribute->getDefaultValue())
                : [];
```

I think it's not obvious to have an empty string in the array of default values. 
I see later it's used with `in_array`, so I think it's better to use empty array instead

Reply:
Hello @ihor-sviziev 
Thank you for your review.
The changes have been added according to previous behaviour of PHP (<= 8.0)
The `explode` of null retrieved array with 1 element `''`.
Please take a look [this example](https://3v4l.org/5YYeX)

Reply:
```
} else {
             $defaultValues = [];
```
Makes me thinking that `[]` is what we really need

Reply:
@xmav , @ihor-sviziev Thanks for your suggestions! I changed my PR, please check.